src/main/java/mate/academy/spring/validation/EmailValidator.java

```diff
@@ -0,0 +1,18 @@
+package mate.academy.spring.validation;
+
+import java.util.regex.Pattern;
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+public class EmailValidator implements ConstraintValidator<Email, String> {
+    private static final String EMAIL_PATTERN = "^[a-zA-Z0-9.!#$%&'*+/=?^_'{|}~-]"
+            + "+@((\\[[0-9]{1,3}\\.[0-9]{1,3}.\\[[0-9]{1,3}."
+            + "\\[[0-9]{1,3}])|(([a-zA-Z\\-0-9]+\\.)+[a-zA-Z]{2,}))$";
+
+    @Override
+    public boolean isValid(String email,
+                           ConstraintValidatorContext constraintValidatorContext) {
+        return email != null
+                && Pattern.compile(EMAIL_PATTERN).matcher(email).matches();
```

Comment:
It is a good practice to compile patterns only once in such services. Let's create a constant of a type Pattern but not a String